pkg/kubectl/cmd/create_configmap.go

```diff
@@ -71,6 +77,7 @@ func NewCmdCreateConfigMap(f cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ConfigMapV1GeneratorName)
 	cmd.Flags().StringSlice("from-file", []string{}, "Key files can be specified using their file path, in which case a default name will be given to them, or optionally with a name and file path, in which case the given name will be used.  Specifying a directory will iterate each named file in the directory that is a valid configmap key.")
 	cmd.Flags().StringArray("from-literal", []string{}, "Specify a key and literal value to insert in configmap (i.e. mykey=somevalue)")
+	cmd.Flags().String("from-env-file", "", "Specify an env file which contains key=val pairs that are inserted into a configmap.")
```

Comment:
Reading the proposal, I think this should make it clear that multiple key=val pairs can be specified in lines, and that Docker .env is an example of valid file format. I'd suggest something like `Specify the path to a file from which to read lines of key=val pairs to create as configmaps (i.e. a Docker .env file)`.